tests/conftest.py

```diff
@@ -0,0 +1,73 @@
+import os
+import subprocess
+import sys
+
+import pytest
+
+# from dotenv import load_dotenv
```

Comment:
Let's minimize this file be removing everything we don't use. I think in the end its just the `assert_command_succeeds` definition, correct? If we need to add stuff in the future we can add the sessionstart and others.